### PR TITLE
Fixed-Billing/Shipping Address edit form design update from order backend 

### PR DESCRIPTION
**Pull request by Kush2507**
### Fixed Issues (if relevant)

1. magento/magento2#23481: Billing/Shipping Address edit form design update from order backend

### Manual testing scenarios (*)

1. Admin -> Sales -> Invoices -> Click on View -> Address Information section -> Billing Address/Shipping Address click on Edit

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/design/adminhtml/Magento/backend/web/css/styles-old.less [hunk 1]
```diff
@@ -5200,3 +5200,8 @@
         page-break-after: avoid;
     }
 }
+
+.form-order-info {
+    max-width: 700px;
```
Comment (sivaschenko) [inline app/design/adminhtml/Magento/backend/web/css/styles-old.less]: Can you please move the styles update to Magento_Sales module styles in black and luma theme and use a variable for max-width if possible

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Hi @Kush2507 thanks for the pull request! Please take a look at my review comments

File: app/design/adminhtml/Magento/backend/web/css/styles-old.less (hunk 1 above)
Comment (Kush2507) [reply app/design/adminhtml/Magento/backend/web/css/styles-old.less]: Hi @sivaschenko 
I have updated the changes. Can you please review?

File: app/design/adminhtml/Magento/backend/Magento_Sales/web/css/source/module/_order.less [hunk 2]
```diff
@@ -33,6 +33,7 @@
 @order-create-icon-refresh__content: @icon-refresh__content;
 @order-create-icon-plus__content: @icon-plus__content;
 @order-create-icon-plus__hover__color: @color-very-dark-gray-black2;
+@order-address-info-width: 700px;
```
Comment (sivaschenko) [inline app/design/adminhtml/Magento/backend/Magento_Sales/web/css/source/module/_order.less:36]: Are there any existing variables to use for the order info width to make it's width relative to related areas if possible?

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for updates @Kush2507 , please see my review comment

**Comment by sidolov** [comment]
@Kush2507 , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Pull request closed without merging**